app/controllers/matches_controller.rb

```diff
@@ -31,25 +30,22 @@ def delete
 
   private
 
-  def adoption_params
-    params.permit(:pet_id, :adopter_account_id, :match_id)
+  def match_params
+    params.require(:match).permit(:pet_id, :adopter_account_id)
   end
 
-  # set status on all applications for a pet
-  def set_statuses_to_adoption_made
-    @applications = Pet.find(params[:pet_id]).adopter_applications
-    @applications.each do |app|
-      unless app.status == "withdrawn"
-        app.status = "adoption_made"
-        app.save
-      end
-    end
+  def set_pet
+    @pet = Pet.find(match_params[:pet_id])
+  end
+
+  def set_match
+    @match = Match.find(params[:id])
   end
 
   def get_pet_id
     return params[:pet_id] if params[:pet_id]
```

Comment:
I _think_ this should be `match_params[:pet_id]`?
```suggestion
    return match_params[:pet_id] if match_params[:pet_id]
```